### PR TITLE
FIX: feature ESC_SENSOR

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: ESC_SENSOR

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]
* enabled motor_stop, esc_sensor via uart, 3d on fresh flash. (some of the following are defaults, but still show in diff)
```
# diff

# version
# Betaflight / STM32F7X2 (S7X2) 4.5.0 Mar 18 2023 / 21:08:01 (ef3ee2eb1) MSP API: 1.46

# start the command batch
batch start

board_name TMOTORF7
manufacturer_id TMTR

# feature
feature MOTOR_STOP
feature TELEMETRY
feature 3D
feature LED_STRIP
feature OSD
feature ESC_SENSOR

# serial
serial 2 1024 115200 57600 0 115200

# master
set osd_canvas_width = 30
set osd_canvas_height = 13

profile 0

rateprofile 0

# end the command batch
batch end
```

**Review by blckmn — APPROVED** [review]

**Pull request merged**
